src/ceph.in

```diff
@@ -74,9 +74,13 @@ from ceph_argparse import \
     send_command, json_command
 
 from ceph_daemon import DaemonWatcher, admin_socket
+from collections import defaultdict
 
 # just a couple of globals
 
+# in case some errno is not recognized.
+errorcode = defaultdict(lambda: 'N/A')
```

Comment:
Perhaps "Unknown" would be a more literal statement here?  N/A to me sounds like we're saying an error doesn't matter.